src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary_tab.cljs

```diff
@@ -24,177 +26,321 @@
-   :persist-acl
-   (fn [{:keys [props state]}]
-     (swap! state assoc :saving? true)
-     (endpoints/call-ajax-orch
-       {:endpoint (endpoints/update-workspace-acl (:workspace-id props))
-        :headers {"Content-Type" "application/json"}
-        :payload (filter #(not (empty? (:userId %))) (:acl-vec @state))
-        :on-done (fn [{:keys [success? status-text]}]
-                   (swap! state dissoc :saving?)
-                   (if success?
-                     ((:dismiss-self props))
-                     (js/alert "Error saving permissions: " status-text)))}))
-   :component-did-mount
-   (fn [{:keys [props state]}]
-     (endpoints/call-ajax-orch
-       {:endpoint (endpoints/get-workspace-acl (:workspace-id props))
-        :on-done (fn [{:keys [success? get-parsed-response status-text]}]
-                   (if success?
-                     (let [acl-vec (build-acl-vec ((get-parsed-response) "acl"))]
-                       (swap! state assoc :acl-vec acl-vec :count-orig (count acl-vec)))
-                     (swap! state assoc :error status-text)))}))})
+               (:attrs-list @state)))])
+        (style/create-paragraph [:em {} "There are no attributes to display"]))]))])
+
+(defn- render-sidebar [state refs props ws viewing-attributes? editing-attributes? owner? writer? status this]
+  [:div {:style {:margin "45px 25px"}}
+   [:div {:style {:float "left" :width 290 :marginRight 40}}
+    (when-not viewing-attributes?
+      [:div {:style {:float "left" :width 290 :marginRight 40}}
+       ;; TODO - make the width of the float-left dynamic
+       [comps/StatusLabel {:text (capitalize (name status))
+                           :icon (case status
+                                   "Complete" [comps/CompleteIcon {:size 36}]
+                                   "Running" [comps/RunningIcon {:size 36}]
+                                   "Exception" [comps/ExceptionIcon {:size 36}])
+                           :color (style/color-for-status status)}]
+       [comps/SidebarButton {:style :light :margin :top :color :button-blue
+                             :text "Edit this page" :icon :pencil
+                             :onClick #(utils/rlog "TODO: implement edit")}]
+       [comps/SidebarButton {:style :light :margin :top :color :button-blue
+                             :text "View attributes" :icon :document
+                             :onClick #(swap! state assoc :viewing-attributes? true)}]
+       (when owner?
+         [comps/SidebarButton {:style :light :margin :top :color :exception-red
+                               :text "Delete" :icon :trash-can
+                               :onClick #(when (js/confirm "Are you sure?")
+                                          (swap! state assoc :deleting? true)
+                                          (react/call :delete this))}])])
+
+    (when (and viewing-attributes? (not editing-attributes?))
+      [:div {}
+       (when writer?
```

Comment:
Is writer the correct access level to require for editing workspace attributes? Or is this something for owners?